### PR TITLE
fix 2022

### PR DESCRIPTION
**Pull request by litewarp**
# Description

Fix for #2022 

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [ ] Documentation has been updated
- [X] TCK tests have been updated
- [X] Integration tests have been updated
- [ ] Example applications have been updated
- [X] New files have copyright header
- [X] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Made some changes to the GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at
 http://neo4j-graphql-toolbox-stage-2023.surge.sh (=> for local development; use for instance [Neo4j Desktop](https://neo4j.com/download/))
or
 https://neo4j-graphql-toolbox-stage-2023.surge.sh (=> use to connect to an [Aura Free](https://neo4j.com/cloud/aura-free/) dbms)

**Review by tbwiss — APPROVED** [review]
LGTM! Thank you @litewarp for testing the fix to https://github.com/neo4j/graphql/issues/1536 so thoroughly!

File: packages/graphql/tests/integration/issues/2022.test.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+ */
+
+import type { GraphQLSchema } from "graphql";
+import { graphql } from "graphql";
+import type { Driver, Session } from "neo4j-driver";
+import Neo4j from "../neo4j";
+import { Neo4jGraphQL } from "../../../src";
+import { generateUniqueType } from "../../utils/graphql-types";
+
+describe("https://github.com/neo4j/graphql/issues/1536_2", () => {
```
Comment (angrykoala) [inline packages/graphql/tests/integration/issues/2022.test.ts:27]: We should probably link to the issue of this problem
```suggestion
describe("https://github.com/neo4j/graphql/issues/2022", () => {
```

File: packages/graphql/tests/tck/tck-test-files/issues/2022.test.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { gql } from "apollo-server";
+import type { DocumentNode } from "graphql";
+import { Neo4jGraphQL } from "../../../../src";
+import { formatCypher, translateQuery, formatParams } from "../../utils/tck-test-utils";
+
+describe("https://github.com/neo4j/graphql/issues/1536", () => {
```
Comment (angrykoala) [inline packages/graphql/tests/tck/tck-test-files/issues/2022.test.ts:25]:
```suggestion
describe("https://github.com/neo4j/graphql/issues/2022", () => {
```

**Review by angrykoala — CHANGES_REQUESTED** [review]
This was an oversight on my part, thanks for fixing it and thoroughly testing it.

It is almost good to go, just a change in the names of the tests to match the issue and we can get it merged

**Pull request merged**